src/clj/game/cards/programs.clj

```diff
@@ -598,20 +598,20 @@
     :abilities [{:cost [:credit 1] :msg (msg "break 1 " (:subtype-target card) " subroutine")}]}
 
    "Chisel"
-   {:effect (effect (system-say "Corp, please press No Action to trigger Chisel"))
+   {:implementation "Encounter effect is manual."
     :hosting {:req (every-pred ice? can-host?)}
+    :abilities [{:req (req (= (:cid current-ice) (:cid (:host card))))
+                 :msg (msg (if (zero? (get-strength current-ice))
+                             (str "trash " (card-str state current-ice))
+                             (str "place 1 virus counter on " (card-str state card))))
```

Comment:
Might want to change to using a `system-msg` for the counter placement.

Reply:
Why? The effect is a firm OR: either ice strength is > 0 so you place a counter OR ice strength is <= 0 so you trash it.

Reply:
So it doesn't print the clunky "Runner uses Chisel to place 1 virus counter on Chisel."
In general we print "Runner places 1 virus counter on Chisel." instead for these kind of effects (I think)